webapp/components/test/test-run.html

```diff
@@ -60,7 +60,7 @@
     });
 
     test('timeFormat(iso string)', () => {
-      assert.equal(trf.timeFormat('2018-01-12T12:00:00Z'), 'Jan 12 at 12:00 PM');
+      assert.equal(trf.timeFormat('2018-01-12T12:00:00Z'), 'Jan 12 at 12:00 PM');
```

Comment:
Could you try to revert this line and let it run?

Reply:
Good idea!

Reply:
Work like a charm

Reply:
So definitely a puppeteer issue